README.md

```diff
@@ -20,9 +20,9 @@ Wagtail-TreeModelAdmin is an extension for Wagtail's [ModelAdmin](http://docs.wa
 
 ## Dependencies
 
-- Python 3.6, 3.7, 3.8
+- Python 3.6, 3.8
 - Django 1.11, 2.0, 2.2
-- Wagtail 1.13, 2.3, 2.7
+- Wagtail 1.13, 2.3, 2.8
```

Comment:
Similar to the comments Betsy and I made on cfpb/wagtail-flags#43, I would give these as

```
- Python 3
- Django 1.11, 2.0–2.2
- Wagtail 1.13, 2.3–2.8
```